release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -99,9 +99,373 @@ Warning: short name "ex" could also match lower priority resource examples.test.
+
+The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:
+
+* Agent-based Installer ISO boot
+* Ingress Node Firewall Operator
+* Secure boot
+
 [id="ocp-4-16-ibm-power"]
 === {ibm-power-title}
 
+{ibm-power-name} is now compatible with {product-title} {product-version}. For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on {ibm-power-name}]
+* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on {ibm-power-name} in a restricted network]
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}.
+====
+
+[discrete]
+==== {ibm-power-title} notable enhancements
+
+The {ibm-power-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.
+
+This release introduces support for the following features on {ibm-power-name}:
+
+* CPU manager
+* Ingress Node Firewall Operator
+* Installer-provisioned Infrastructure for IBM Power® Virtual Server - move to CAPI
+
+[discrete]
+=== {ibm-power-title}, {ibm-z-title}, and {ibm-linuxone-title} support matrix
+
+Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibm-power-name} and the {ibm-z-name} platform. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
+
+.{product-title} features
+[cols="3,1,1",options="header"]
+|====
+|Feature |{ibm-power-name} |{ibm-z-name} and {ibm-linuxone-name}
+
+|Alternate authentication providers
+|Supported
+|Supported
+
+|Agent-based Installer
+|Supported
+|Supported
+
+|Assisted Installer
+|Supported
+|Supported
+
+|Automatic Device Discovery with Local Storage Operator
+|Unsupported
+|Supported
+
+|Automatic repair of damaged machines with machine health checking
+|Unsupported
+|Unsupported
+
+|Cloud controller manager for {ibm-cloud-name}
+|Supported
+|Unsupported
+
+|Controlling overcommit and managing container density on nodes
+|Unsupported
+|Unsupported
+
+|Cron jobs
+|Supported
+|Supported
+
+|Descheduler
+|Supported
+|Supported
+
+|Egress IP
+|Supported
+|Supported
+
+|Encrypting data stored in etcd
+|Supported
+|Supported
+
+|FIPS cryptography
+|Supported
+|Supported
+
+|Helm
+|Supported
+|Supported
+
+|Horizontal pod autoscaling
+|Supported
+|Supported
+
+|Hosted control planes (Technology Preview)
+|Supported
+|Supported
+
+|IBM Secure Execution
+|Unsupported
+|Supported
+
+|{ibm-power-server-name} Block CSI Driver Operator
+|Supported
+|Unsupported
```

Comment:
@alishaIBM I think this should be moved to the Operators table

```suggestion
|{ibm-power-server-name} Block CSI Driver Operator
|Supported
|Unsupported
```

Reply:
Agree. Moved to the `Operators` section. Thanks.